drv/transceivers-server/src/udp.rs

```diff
@@ -1039,8 +1191,17 @@ impl ServerImpl {
                 userlib::hl::sleep_for(1);
             }
         }
-        let final_result = ModuleResult::new(success, failure, error).unwrap();
-        result.chain(final_result)
+        let mut final_result =
+            ModuleResult::new(success, failure, error, failure_types).unwrap();
```

Comment:
i think this is, thankfully, the last one
```suggestion
            ModuleResult::new(success, failure, error, failure_types).unwrap_lite();
```